### PR TITLE
add note about /png

### PR DESCRIPTION
**Pull request by tardyp**

### CODE REVIEW

File: master/docs/relnotes/index.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Release Notes for Buildbot |version|
     Any change that adds a feature or fixes a bug should have an entry here.
     Most simply need an additional bulleted list item, but more significant
     changes can be given a subsection of their own.
+    If you can:
+      please point to the bug using syntax: (:bug:`NNN`)
+      please point to classes uing syntax :py:class:`~buildbot.reporters.http.HttpStatusBase`
```
Comment (rutsky) [inline master/docs/relnotes/index.rst]: Typo: `uing`

File: master/docs/relnotes/index.rst [hunk 2]
```diff
@@ -36,9 +39,15 @@ Fixes
 Deprecations, Removals, and Non-Compatible Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* GitLabStatus was present in buildbot 0.8.12 release, but is no more in 0.9.0.
+* GitLabStatus was present in buildbot 0.8.12 release, but is no more in 0.9.0 (:bug:`3286`).
   Gitlab community should be able to easily port it using :py:class:`~buildbot.reporters.http.HttpStatusBase`
 
+
+* Web server does not provide /png and /redirect anymore (:bug:`3357`).
+  This function is used to implement build status images.
```
Comment (rutsky) [inline master/docs/relnotes/index.rst]: What is "this function" here?

File: master/docs/relnotes/index.rst [hunk 3]
```diff
@@ -36,9 +39,15 @@ Fixes
...
+  This should be easy to implement it you need it, porting the old method, or redirecting to http://shields.io/.
```
Comment (rutsky) [inline master/docs/relnotes/index.rst]: Something missed here, maybe "**if** you need it"?

File: master/docs/relnotes/index.rst (hunk 1 above)
Comment (djmitche) [inline master/docs/relnotes/index.rst]: This also needs .rst list formatting

File: master/docs/relnotes/index.rst (hunk 2 above)
Comment (tardyp) [inline master/docs/relnotes/index.rst]: functionality

File: master/docs/relnotes/index.rst (hunk 1 above)
Comment (tardyp) [inline master/docs/relnotes/index.rst]: it is a rst comment

**Comment by tardyp** [comment]
fixed as per review

**Comment by rutsky** [comment]
Here is how changes looks like: https://circle-artifacts.com/gh/buildbot/buildbot/370/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/relnotes/index.html

LGTM

**Pull request merged**